invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -162,3 +169,19 @@ $('#licenses .licenses-description .close.icon')
       }
     }
   });
+
+
+  // Record management popup (mobile)
+
+  $('#manage-record-btn')
```

Comment:
shouldn't the accesibility attrs be applied to all the popup elements not only the record management?

Reply:
Ref comment: https://github.com/inveniosoftware/invenio-app-rdm/pull/1403#discussion_r848096050